lib/individual/info_page.dart

```diff
@@ -24,7 +25,7 @@ class _IndividualInfoPageState extends State<IndividualInfoPage> {
               color: Colors.black,
               size: MediaQuery.of(context).size.height * 0.025),
           leading: IconButton(
-            onPressed: () => Navigator.of(context).pop(),
+            onPressed: () => Navigator.push(context, MaterialPageRoute(builder: (context) => const HomePageIndividual())),
```

Comment:
This concept of pushing the Home Page upon clicking the Back Arrow button, does fix the issue. But it also includes many performance issues :thinking: . For now, this is alright, but we will need to change this to popUntil when we implement named-routing in our application.